src/tls_wrap.cc

```diff
@@ -401,6 +400,9 @@ void TLSWrap::ClearOut() {
   if (ssl_ == nullptr)
     return;
 
+  crypto::ClearErrorOnReturn clear_error_on_return;
+  (void) &clear_error_on_return;  // Silence compiler warning.
```

Comment:
I think you can drop this line now.  IIRC, that was a workaround for a false positive in gcc 4.2 or 4.4.